### PR TITLE
cleanup(samples): Samples moved to python-docs-samples repo

### PR DESCRIPTION
**Pull request by m-strzelczyk**
The samples have been moved here: https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/compute/client_library

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -1,53 +0,0 @@
-# Code samples for the Compute Engine library
```
Comment (dandhlee) [inline samples/README.md:1]: Could you add in a README here to let folks know that the samples have migrated:

```
Samples migrated
================

New location: https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/compute/client_library
```

**Review by dandhlee — COMMENTED** [review]
LGTM, one minor nit.

File: samples/README.md (hunk 1 above)
Comment (m-strzelczyk) [reply samples/README.md:1]: Sure, good idea :)

**Review by dandhlee — APPROVED** [review]
File numbers match, LGTM.

**Pull request merged**